src/interface/TWEthereumAbiFunction.cpp

```diff
@@ -74,6 +74,16 @@ int TWEthereumAbiFunctionAddParamUInt64(struct TWEthereumAbiFunction *_Nonnull f
     return idx;
 }
 
+int TWEthereumAbiFunctionAddParamUInt64SkipEncode(struct TWEthereumAbiFunction *_Nonnull func_in, uint64_t val, bool isOutput) {
```

Comment:
Did you only add this function in here to simplify testing? In that case I think we should try to push it into the tests.

Reply:
This is used only in tests, but it could be used by any consumer library. Same applies to all other pre-existing `TWEthereumAbiFunctionAddParam...` methods.